src/test/java/com/org/jenkins/custom/jenkins/distribution/service/SpringMVCSetup.java

```diff
@@ -14,6 +15,7 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest(classes = CustomJenkinsDistributionServiceApplication.class)
 @WebAppConfiguration
+@AutoConfigureMockMvc
```

Comment:
Is there a test that we want to update to go with importing this into this class?

Reply:
Not that i know of